src/modules/launcher/Plugins/Microsoft.Plugin.WebSearch/Languages/de.xaml

```diff
@@ -0,0 +1,33 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:system="clr-namespace:System;assembly=mscorlib">
+
+    <system:String x:Key="wox_plugin_websearch_delete">Löschen</system:String>
```

Comment:
Should this be changed to microsoft_plugin... ?

Reply:
Good catch. I will change it.